core/src/main/java/org/mapfish/print/URIUtils.java

```diff
@@ -157,7 +157,7 @@ private static boolean addParams(
         result.append("=");
         result.append(URLEncoder.encode(val, Constants.DEFAULT_ENCODING));
       } catch (UnsupportedEncodingException e) {
-        throw ExceptionUtils.getRuntimeException(e);
+        throw new PrintException("Failed to encode using " + Constants.DEFAULT_ENCODING, e);
```

Comment:
No need to add val?

Reply:
it is not always val failing. So I changed the code to report the correct issue.